### PR TITLE
#49 save places to favorites

### PR DESCRIPTION
**Pull request by ZakharSkaletskyi**

### CODE REVIEW

File: src/main/java/greencity/repository/FavoritePlaceRepo.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
      * @param email   - user's email
      * @author Zakhar Skaletskyi
      */
-    void deleteByPlaceIdAndUserEmail(Long placeId, String email);
+    int deleteByPlaceIdAndUserEmail(Long placeId, String email);
```
Comment (MarianMilian) [inline src/main/java/greencity/repository/FavoritePlaceRepo.java]: deleteByPlaceIdAndUserEmail() method shoud return Long

Comment (ZakharSkaletskyi) [reply src/main/java/greencity/repository/FavoritePlaceRepo.java]: Ids return not id

File: src/main/java/greencity/service/impl/UserServiceImpl.java [hunk 2]
```diff
@@ -104,17 +105,11 @@ public User findByEmail(String email) {
...
-
-    /**
-     * {@inheritDoc}
-     *
-     * @author Zakhar Skaletskyi
-     */
-    @Override
-    public boolean existsByEmail(String email) { //zakhar
-        return repo.existsByEmail(email);
+        Long id = repo.findIdByEmail(email);
```
Comment (MarianMilian) [inline src/main/java/greencity/service/impl/UserServiceImpl.java]: Probably  you  shoud check on  null

Comment (ZakharSkaletskyi) [reply src/main/java/greencity/service/impl/UserServiceImpl.java]: email?

Comment (MarianMilian) [reply src/main/java/greencity/service/impl/UserServiceImpl.java]: not email, next line

**Pull request closed without merging**